internal/cloudinfo/machine_types.go

```diff
@@ -94,39 +96,42 @@ func (im *InstanceTypeMap) setMachines(cloud string, service string, region stri
 var instanceTypeMap = NewInstanceTypeMap()
 
 func fetchMachineTypes(logger logrus.FieldLogger, cloud string, service string, region string) error {
-	cloudInfoEndPoint := viper.GetString(config.CloudInfoEndPoint)
-	if len(cloudInfoEndPoint) == 0 {
-		return emperror.With(errors.New("missing config"), "propertyName", config.CloudInfoEndPoint)
-	}
-	cloudInfoUrl := fmt.Sprintf(
-		"%s/providers/%s/services/%s/regions/%s/products",
-		cloudInfoEndPoint, cloud, service, region)
-	ciRequest, err := http.NewRequest(http.MethodGet, cloudInfoUrl, nil)
-	if err != nil {
-		return errors.New("error fetching machine types from CloudInfo")
+	ciHostName := viper.GetString(config.CloudInfoHostName)
+	if len(ciHostName) == 0 {
+		return emperror.With(errors.New("missing config"), "ciHostName", config.CloudInfoHostName)
```

Comment:
I don't think ciHostName is good name for the context, I would use fmt.Errorf("missing config: %s", config.CloudInfoHostName) instead of just errors.New and a context.

Reply:
renamed